CHANGELOG.md

```diff
@@ -21,21 +23,21 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 - Fix monitored file system mount points.
 
-## [v0.3.0]
+## [0.3.0] 2019-05-24
 
 ### Changed
 
 - Upgrade to node-exporter 0.18.0.
 
-## [v0.2.0]
+## [0.2.0] 2019-05-17
 
 ### Added
 
 - Separate pod security policy for node-exporter and node-exporter-migration workloads.
 - Security context with non-root user (`nobody`) for running node-exporter inside container.
 
 [Unreleased]: https://github.com/giantswarm/kubernetes-node-exporter/compare/v0.4.1...HEAD
-[0.4.1]: https://github.com/giantswarm/kubernetes-node-exporter/releases/tag/v0.4.1
-[0.4.0]: https://github.com/giantswarm/kubernetes-node-exporter/releases/tag/v0.4.0
-[0.3.0]: https://github.com/giantswarm/kubernetes-node-exporter/releases/tag/v0.3.0
+[0.4.1]: https://github.com/giantswarm/kubernetes-node-exporter/compare/v0.4.0...v0.4.1
+[0.4.0]: https://github.com/giantswarm/kubernetes-node-exporter/compare/v0.3.0...v0.4.0
+[0.3.0]: https://github.com/giantswarm/kubernetes-node-exporter/compare/v0.2.0...v0.3.0
 [0.2.0]: https://github.com/giantswarm/kubernetes-node-exporter/releases/tag/v0.2.0
```

Comment:
I pushed tags manually. That doesn't hurt and changelogs give better overview.

Reply:
Why are the links different though? Shouldn't it be all compare or all tags?

Reply:
This is the same way as in https://raw.githubusercontent.com/olivierlacan/keep-a-changelog/master/CHANGELOG.md.

Reply:
I still do not understand it. What is the reasoning or scheme here?

Reply:
Did you try both? To show all commits/changes included in the release instead of the last commit.

See e.g.:
- Old: https://github.com/giantswarm/kubernetes-node-exporter/releases/tag/v0.3.0
- New: https://github.com/giantswarm/kubernetes-node-exporter/compare/v0.2.0...v0.3.0

Reply:
Is it just that the very first tag has no tag to compare against and that this is different because of this circumstance?

Reply:
This I can't tell. I just copied the approach in the example.